docs/eventing/debugging/example.yaml

```diff
@@ -95,9 +95,7 @@ spec:
     spec:
       containers:
         - name: user-container
-          # This corresponds to
-          # https://github.com/knative/eventing-contrib/blob/v0.2.1/cmd/message_dumper/dumper.go.
-          image: gcr.io/knative-releases/knative.dev/eventing-sources/cmd/message_dumper
+          image: gcr.io/knative-releases/knative.dev/eventing-sources/cmd/event_display
```

Comment:
Ok looking for this image and I don't see that it actually exists. Need a different URL. I'll ask around.

Reply:
It's actually `gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display`

There are a few other places where the wrong url is used:
https://github.com/search?q=org%3Aknative+eventing-sources+event_display&type=Code
https://github.com/search?q=org%3Aknative-sandbox+eventing-sources+event_display&type=Code

Reply:
I took a look and it looks like we use:

`gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display`

elsewhere in the [docs](https://knative.dev/docs/eventing/samples/container-source/#create-a-knative-service), so maybe either that, or we need to figure out what the working one is and use it consistently?

Reply:
Haha snap was just saying that :smile: I'll take another stab at fixing it

Reply:
I'll leave the knative-sandbox repo up to engineering folks to fix since this is just a docs issue/PR and I don't mess with that usually :slightly_smiling_face: